shared/constants/fs.js

```diff
@@ -287,8 +288,7 @@ export const downloadFilePathFromPathNoSearch = (p: Types.Path): string =>
   downloadFilePathNoSearch(Types.getPathName(p))
 
 export const isImage = (name: string): boolean => {
-  const lower = name.toLowerCase()
-  return ['.png', '.jpg', '.jpeg', '.mp4'].some(ext => lower.endsWith(ext))
+  return (mime.lookup(name) || '').indexOf('image/') !== -1
```

Comment:
Nice! Perhaps include `video/` and `audio/` too? Perhaps this function should've been named `isMedia`.